client/src/main/java/io/split/client/api/SplitView.java

```diff
@@ -16,4 +14,5 @@
     public boolean killed;
     public List<String> treatments;
     public long changeNumber;
+    public Map<String, String> config;
```

Comment:
This should be plural because it will have the configurations for all the treatments, so it should be `configs` instead of `config`